src/core.ts

```diff
@@ -84,14 +84,13 @@ function getQueryInterface(
 }
 
 function generate(routes: Route[]): string {
+  const unknownQueryParamsType =
+    "{ [key: string]: string | string[] | undefined }";
```

Comment:
👍 This is a nice way to control the exported namespaces while still cutting down on duplication